src/components/pages/wallets/casestudies.tsx

```diff
@@ -0,0 +1,311 @@
+import classnames from 'classnames'
+import { withPrefix } from 'gatsby'
+import React, { useState, useCallback, useMemo, Fragment } from 'react'
+
+import GoLeft from '../../../assets/wallets/casestudies/left.svg'
+import OrangeMark from '../../../assets/wallets/casestudies/orangemark.svg'
+import GoRight from '../../../assets/wallets/casestudies/right.svg'
+import Button from '../../common/button'
+import Wave from '../../common/wave'
+
+import casestudies from './caseStudyData'
+import pressdata from './pressData'
+
+type CompanyChipProps = {
+  active: boolean
+  Logo: React.FunctionComponent<React.SVGAttributes<SVGElement>>
+}
+
+const CompanyChip: React.FC<CompanyChipProps> = ({ active, Logo }) => {
+  return (
+    <Fragment>
+      <div
+        className={`${
+          active ? 'bg-gradient-r-orange-dark-to-light' : ''
+        } ml-2 mr-2 lg:w-48 w-30 h-1 mb-3`}
+      />
+      <div className="flex items-center mt-1 ml-2 mr-2 bg-white rounded cursor-pointer lg:w-48 w-38 h-18">
+        <Logo className="w-11/12 m-auto" />
+      </div>
+    </Fragment>
+  )
+}
+
+type CompanyChipRowProps = {
+  chips: Array<{
+    logo: React.FunctionComponent<React.SVGAttributes<SVGElement>>
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [key: string]: any
```

Comment:
Can we refactor this to not be an `any`? Can we not cast this as a `string`?

Reply:
@Stormtv haven't been able to get this working. It complains that the `logo` key is not a string. It seems that you need to make all keys the same if you use this syntax.